GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+- [Modifying this Charter](#modifying-this-charter)
+<!-- TOC end -->
+
+## Values
+
+The RIOT community embraces the following values:
+
+* **Openness:** Communication and decision-making happens in the open and is discoverable for future reference.
+  As much as possible, all discussions and work events eventually take place in public forums and open repositories.
+
+* **Fairness:** All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits.
+
+* **Community over Product or Company:** Sustaining and growing our community takes
+  priority over shipping code or sponsors' organizational goals.
+  Each contributor participates in the project as an individual.
+
+* **Inclusivity:** We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+
+* **Participation:** Responsibilities within the project are earned through participation, and there is a clear path up the contributor ladder into more responsible positions.
+
+
+## Community Processes
+The community around RIOT gathers many IoT developers and users from around the world, from the industry, from academia, and hobbyists.
+The RIOT community is open to everyone.
+Our channels to communicate can be found in our [contributing guidelines].
+The community self-organizes using the roles described below.
+
+
+## Roles
+### Contributors
+Contributors are people who contribute their work to RIOT.
+This includes,
+
+- of course, code contributions, but also
+- bug reports,
+- other issue reports,
+- participation in technical as well as non-technical discussions, or
+- organizational considerations.
+
+Code contributions are very welcome.
+In order to streamline and harmonize code quality, contributors must follow the [contributing guidelines].
+
+
+### Maintainers
+
+Among contributors, some have maintainer status, which consists in rights (write access to the [RIOT GitHub repository](https://github.com/RIOT-OS/RIOT/)) and duties.
+The current maintainers can be found in the [maintainers list].
+
+Maintainers are people who care about RIOT and want to help it grow and improve.
+A maintainer is not just someone who can make changes, but someone who has demonstrated their ability to collaborate with the team, get the most knowledgeable people to review code and docs, contribute high-quality code, and follow through to fix issues (in code or tests).
+More on maintaining RIOT can be found in the [maintaining guidelines].
+
+We are constantly looking for more maintainers.
+So if you are up for that, please start (or continue) contributing code and reviews!
+
+To contact maintainers, the best is to interact over actual RIOT code on [GitHub](https://github.com/RIOT-OS/RIOT/pulls).
+
+#### Becoming a Maintainer
+
+Maintainers can propose to give maintainer status to contributors that have been noticed as particularly active in some domain of RIOT.
```

Comment:
In the past it was very obvious in their behavior if a contributor intended to be a maintainer (reviewing PRs even without the rights to merge and writing issues), but in general there is no precedence of a would-be maintainer outright asking. So I'd say let's keep it as is, but I will put this in my forum summary of the current state of discussion.

Reply:
Hmm, let's say a contributor would approach a maintainer to also become a maintainer. I would assume that this would then be raised to the other maintainers and discussed pretty much in the same way as if the a maintainer suggested it.

Or would anyone of us reject such a request right away without reaching out to the others?

If not, we could also IMO just document that contributors may reach out to a maintainer of their choice, who will forward this to the other maintainers for consideration.

Reply:
> If not, we could also IMO just document that contributors may reach out to a maintainer of their choice, who will forward this to the other maintainers for consideration.

Question is: is this something we need to put in our governance document, i.e., do we want to make it “law” that this is a way to become a maintainer? This could go into the contribution ladder document @jkarinkl proposed and would probably be a better fit.

Reply:
I would recommend keeping the governance doc as short as possible, so indeed, add this  to the contribution ladder. This part is about being friendly and welcoming to enthusiastic contributors, making it easy to step up if they are capable and willing.

Reply:
> This part is about being friendly and welcoming to enthusiastic contributors, making it easy to step up if they are capable and willing.

So to add or not to add a sentence?